### PR TITLE
feat: make 'src' configurable in baseUrl

### PR DESCRIPTION
**Pull request by ritingliudd01**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "unimported",
-  "version": "1.20.2",
+  "version": "1.20.4",
```
Comment (smeijer) [inline package-lock.json:3]:
```suggestion
  "version": "1.20.2",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "unimported",
-      "version": "1.20.3",
+      "version": "1.20.4",
```
Comment (smeijer) [inline package-lock.json:9]:
```suggestion
      "version": "1.20.3",
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "unimported",
-  "version": "1.20.2",
+  "version": "1.20.4",
```
Comment (smeijer) [inline package.json:3]:
```suggestion
  "version": "1.20.2",
```

**Review by smeijer — APPROVED** [review]
Thank you! Much appreciated.

**Comment by smeijer** [comment]
@all-contributors please add @ritingliudd01 for code, test

**Comment by ritingliudd01** [comment]
@smeijer Thank you for your great tool and continuous efforts.

**Pull request merged**